modules/otel-collector-components.adoc

```diff
@@ -82,6 +82,55 @@ The Jaeger receiver ingests traces in the Jaeger formats.
 <4> The Jaeger Thrift Binary endpoint. If omitted, the default `+0.0.0.0:6832+` is used.
 <5> The  server-side TLS configuration. See the OTLP receiver configuration section for more details.
 
+[id="kubeletstats-receiver_{context}"]
+=== Kubelet Stats Receiver
+
+The Kubelet Stats receiver is currently a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] feature only.
+
+The Kubelet Stats receiver extracts metrics related to nodes, pods, containers, and volumes from the kubelet's API server. These metrics are then channeled through the metrics-processing pipeline for additional analysis.
+
+.OpenTelemetry Collector custom resource with an enabled Kubelet Stats receiver
+[source,yaml]
+----
+config: |
+  receivers:
+    kubeletstats:
+      collection_interval: 20s
+      auth_type: "serviceAccount"
+      endpoint: "https://${env:K8S_NODE_NAME}:10250"
+      insecure_skip_verify: true
+  service:
+    pipelines:
+      metrics:
+        receivers: [kubeletstats]
+env:
+  - name: K8S_NODE_NAME # <1>
+    valueFrom:
+      fieldRef:
+        fieldPath: spec.nodeName
+----
+<1> Sets the `K8S_NODE_NAME` to authenticate to the API.
+
+The Kubelet Stats receiver requires additional permissions for the service account used for running the OpenTelemetry Collector.
+
+.Permissions required by the service account
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: otel-collector
+rules:
+  - apiGroups: [""]
+    resources: ["nodes/stats"] # <1>
+    verbs: ["get"]
+  - apiGroups: [""]
+    resources: ["nodes/proxy"] # <2>
+    verbs: ["get"]
+----
+<1> Requires the `get` permissions.
+<2> Requires additional permissions when using the `extra_metadata_labels` or the `request_utilization` or `limit_utilization` metrics.
```

Comment:
@iblancasa, about callout 2, `Requires` which additional `permissions`? Do we need to specify which?

Reply:
The permissions are:
```yaml
  - apiGroups: [""]
    resources: ["nodes/proxy"] # <2>
    verbs: ["get"]
```

If you are not using what is described in `<2>`, you don't need permissions over `nodes/proxy`.

Reply:
So, for me it's fine.

Reply:
Thank you, Israel.

Reply:
I rewrote this callout.